internal/registry/regsrc/friendly_host_test.go

```diff
@@ -133,8 +133,8 @@ func TestInvalidHostEquals(t *testing.T) {
 		t.Fatalf("%q is not equal to %q", valid, invalid)
 	}
 
-	puny := NewFriendlyHost("xn--s-fka0wmm0zea7g8b.xn--o-8ta85a3b1dwcda1k.io")
-	display := NewFriendlyHost("ʎɹʇsıƃǝɹ.ɯɹoɟɐɹɹǝʇ.io")
+	puny := newFriendlyHost("xn--s-fka0wmm0zea7g8b.xn--o-8ta85a3b1dwcda1k.io")
+	display := newFriendlyHost("ʎɹʇsıƃǝɹ.ɯɹoɟɐɹɹǝʇ.io")
```

Comment:
Just because this is not found by regular search & replace. You could replace `ɯɹoɟɐɹɹǝʇ` with `nɟoʇuǝdo` which is upsidedown `terraform`/`opentofu`

Reply:
Woah! What a weird string there… 🤯

Reply:
I think this is worthy of a second issue and PR, which I'm happy to take on once this one is merged or rejected.

For the record, `https://ʎɹʇsıƃǝɹ.nɟoʇuǝdo.io` translates to `xn--s-fka0wmm0zea7g8b.xn--noudo-6yb12ggj.io`.

Reply:
I love this suggestion (feel free to do it in a separate PR of course)